api.js

```diff
@@ -56,309 +60,310 @@ Api.prototype._runFile = function (file, runStatus, execArgv) {
 	});
 
 	var emitter = fork(file, options, execArgv);
-
 	runStatus.observeFork(emitter);
 
 	return emitter;
 };
 
+Api.prototype.run = function (files, options) {
+	var self = this;
+
+	var findOptions = {
+		cwd: this.options.resolveTestsFrom,
+		files: files
+	};
```

Comment:
I don't really see how this improves anything though. It was already clear what the options are for.